README.md

```diff
@@ -155,15 +155,13 @@ In general, use a logical process of elimination to find where the fault (or fau
 
 ## Emulation Accuracy
 
-The logic for both the MDA and CGA cards is as close as I could get it to match the schematics available in the technical reference manuals, with two exceptions. The VRAM interface is specific to the 8-bit SRAM chip that I am using instead of the 16-bit SRAM on the original MDA or the 16-bit DRAM used on the CGA, so it had to be quite different from the originals. As a result, the sequencer state machine had to be designed from scratch. There are some other minor differences mostly to support the nonstandard VGA-compatible signal outputs.
+The logic for both the HGC and CGA cards is as close as I could get it to match the schematics available in the technical reference manuals, with two exceptions. The VRAM interface is specific to the 8-bit SRAM chip that I am using instead of the 16-bit SRAM on the original HGC or the 16-bit DRAM used on the CGA, so it had to be quite different from the originals. As a result, the sequencer state machine had to be designed from scratch. There are some other minor differences mostly to support the nonstandard VGA-compatible signal outputs.
```

Comment:
HGC schematics are not in the reference manuals, so maybe leave this at MDA and add a sentence to describe how you added HGC support

Reply:
I have not relied on HGC schematics, but on the technical characteristics of the graphic mode. In fact, I have reused the CGA 640x200x2 rendering mode from their own code. I will soon make the modifications in this pull request.